ptu/global.ini

```diff
@@ -40993,25 +40993,25 @@ TransportGuild_RepUI_Founded,P=WIP Founded
 TransportGuild_RepUI_Headquarters,P=WIP Headquarters
 TransportGuild_RepUI_Leadership,P=WIP Leadership
 TransportGuild_RepUI_Name,P=WIP Transport Guild
-Tut01_Hint01_ExitBed=Dieses Tutorial führt dich durch deine ersten Schritte im Leben unter den Sternen im 30. Jahrhundert.\nUm aus Betten oder Sitzen auszusteigen, drücke [~action(default|pl_exit)].
+Tut01_Hint01_ExitBed=Um von einem Bett oder Sitzplatz aufzustehen, drücke [~action(default|pl_exit)].
 Tut01_Hint01_ExitBed_Title=Willkommen in Star Citizen!
 Tut01_Hint01b_Regen=Herzlichen Glückwunsch! Du bist gestorben. Das ist im 'Verse keine Seltenheit. Während des Tutorials bringt dich der Tod zurück zu einem Habitat, aber nach dem Tutorial erwachst du in einer medizinischen Einrichtung.
 Tut01_Hint01b_Regen_Title=Im 'Verse sterben
 Tut01_Hint02_Movement=Um dich umzusehen, benutze [~action(player|rotatepitch)] und [~action(player|rotateyaw)]. Zum Bewegen verwende [~action(player|moveforward)] [~action(player|moveleft)] [~action(player|moveback)] [~action(player|moveright)].
 Tut01_Hint02_Movement_Title=Steuerung – Grundbewegung
-Tut01_Hint02a_Mobiglas=Dein mobiGlas-Personalcomputer verfügt über ein Journal, in dem nützliche Informationen und Dokumente aufbewahrt werden.\nDrücke [~action(player|mobiglas)], um auf dein mobiGlas zuzugreifen, und wähle aus der unteren Symbolleiste die Journal-App aus.
+Tut01_Hint02a_Mobiglas=Drücke [~action(player|mobiGlas)], um auf dein mobiGlas zuzugreifen. Wähle dann das Journal aus der unteren Reihe der Symbole.
 Tut01_Hint02a_Mobiglas_Title=mobiGlas – Journal
-Tut01_Hint02b_ContractManager=Dein mobiGlas verfügt auch über einen Vertragsmanager, der verfügbare, angenommene und abgeschlossene Verträge anzeigt. Navigiere zum Tab "Angenommen", um mehr Informationen zum aktuellen Tutorial zu sehen.
+Tut01_Hint02b_ContractManager=Dein mobiGlas-Vertragsmanager zeigt verfügbare, angenommene und abgeschlossene Verträge an. Gehe zum Tab "Angenommen", um Informationen zum Tutorial zu erhalten.
 Tut01_Hint02b_ContractManager_Title=mobiGlas – Vertragsmanager
-Tut01_Hint03_HungerThirst=Dein Hunger und Durst werden unten links auf dem Bildschirm verfolgt. Du musst essen und trinken, um gesund zu bleiben.
+Tut01_Hint03_HungerThirst=Hunger und Durst werden unten links auf deinem Bildschirm verfolgt. Iss und trinke, um zu bleiben.
```

Comment:
Ist hier am Ende um zu <...> absichtlich Leer ? Also ohne " Gesund "

Reply:
Jo ^^ original: . Eat and drink to stay 
Ich kleb da mal ein Gesund mit rein